src/Components/ArticlesList.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "./ArticleCard"; 
+
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    axios.get("https://nc-news-ts.onrender.com/api/articles")
```

Comment:
Please extract this axios request into a separate file and use axios.create() to have a base URL. See NC Notes for that.